### DISCUSSION TITLE
Love the plugin, however it's removing Holograms (entities armor_stand), how do I stop that from happening?

### DISCUSSION THREAD

**Discussion by MythionVR** [GriefPrevention/GriefPrevention#893] [Q&A] [answered]
This is the error I get: 
[04:12:21 INFO]: Removing entity ARMOR_STAND @ Location{world=CraftWorld{name=creative},x=-159.42489082270396,y=65.0,z=-103.52300065692445,pitch=0.0,yaw=3.2910402}

However it happens on my Survival world where I need GP enabled as well. I've checked and re-checked the settings for both to see if there was anything I could modify. I'm not sure how I could fix this, if there is one.

Thanks in advance!

**Comment by Jikoo** [comment, marked as answer]
Entity cleanup only happens in worlds using Creative claims mode. You can use Survival or SurvivalRequiringClaims to not enable the creative ruleset.

**Comment by Jikoo** [comment]
Closing due to lack of response. If you have problems in survival worlds, please open an issue using the bug report template.
